.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# How to contribute
+
+Thank you for your interest in contributing to the Aerogear project. We want
+keep this process as easy as possible so we've outlined a few guidelines below. 
+For more information about the Aerogear project, visit 
+[our website](http://aerogear.org/).
+
+## Asking for help
+
+Whether you're contributing a new feature or bug fix, or simply submitting a
+ticket, the Aerogear team is available for technical advice or feedback. 
+You can reach us at #aerogear on [Freenode IRC](https://freenode.net/) or the 
+[aerogear-dev list](http://lists.jboss.org/pipermail/aerogear-dev/)
```

Comment:
we should get rid of aerogear-dev list as we moved away from it. Please mention the Google group though